build-go.sh

```diff
@@ -97,7 +97,7 @@ function goBuild() {
 	(
 		export GOOS="$goos"
 		if [[ "$goarch" == amd64-* ]]; then
-			IFS="-"; read -ar arr <<<"$goarch"
+			IFS="-" read -ra arr <<< "$goarch"
```

Comment:
Bash have special sementics when setting IFS before read on the same line it resets IFS, but that more readable so let's keep it that way.